pom.xml

```diff
@@ -82,6 +80,14 @@
             <groupId>org.osgi</groupId>
             <artifactId>osgi.core</artifactId>
```

Comment:
we should switch to individual OSGi artifacts

Reply:
done